docs/developers/contributing.md

```diff
@@ -29,12 +29,14 @@ Bugfixes should include a regression test which prevents that bug from being re-
 
 ## Update Documentation
 
-Documentation for the project is stored in the [docs](..) directory. If your feature requires docs updates,
-those updates must be submitted in the same Pull Request as the code changes.
+Documentation for the project is stored in the prebid-server directory on the [Prebid repo](https://github.com/prebid/prebid.github.io/tree/master/prebid-server). If your feature requires docs updates,
+those updates must be submitted in the same Pull Request as the code changes. Documentation can be viewed at 
 
 - [docs/endpoints](../endpoints) describes the Prebid Server API. For example, the endpoint `host:port/openrtb2/auction` is described by [docs/endpoints/openrtb2/auction.md](../endpoints/openrtb2/auction.md)
```

Comment:
The links here should to be updated to their appropriate location in the [prebid.github.io](https://github.com/prebid/prebid.github.io) repo.